document_page/views/document_page.xml

```diff
@@ -0,0 +1,219 @@
+                    <field
+                        name="content"
+                        widget="html"
+                        placeholder="e.g. Once upon a time..."
+                        required="1"
+                        options="{'safe': True, 'codeview': True, 'collaborative': True}"
+                    />
+                    <notebook>
+                        <page name="info" string="Information">
+                            <group>
+                                <group>
+                                    <field
+                                        name="parent_id"
+                                        required="True"
+                                        string="Category"
+                                        context="{'default_type':'category', 'form_view_ref': 'document_page.view_category_form'}"
+                                    />
+                                    <field
+                                        name="company_id"
+                                        groups="base.group_multi_company"
+                                    />
+                                    <field name="content_uid" />
+                                    <field name="content_date" />
+                                    <field
+                                        name="menu_id"
+                                        readonly="1"
+                                        invisible="not menu_id"
+                                    />
+                                </group>
+                                <group string="Revision">
+                                    <field
+                                        name="draft_name"
+                                        placeholder="Rev 01"
+                                        required="True"
+                                    />
+                                    <field
+                                        name="draft_summary"
+                                        placeholder="eg: Changed ... for ..."
+                                        required="True"
+                                        widget="text"
+                                    />
+                                </group>
+                            </group>
+                        </page>
+                        <page name="history" string="History">
+                            <field name="history_ids">
+                                <list>
+                                    <field name="id" />
+                                    <field name="create_date" />
+                                    <field name="name" />
+                                    <field name="summary" />
+                                    <field name="create_uid" />
+                                </list>
+                            </field>
+                        </page>
+                    </notebook>
+                </sheet>
+                <div class="oe_chatter">
```

Comment:
It seems like you forgot to replace this with `<chatter />`

Reply:
Thanks for the catch. It's fixed now